hadoop-cloud/pom.xml

```diff
@@ -201,7 +201,7 @@
      enables store-specific committers.
     -->
     <profile>
-      <id>hadoop-3.2</id>
+      <id>hadoop-3.3</id>
```

Comment:
Same comment as hive-2.3, but, here may be better to change the name to make sure callers are clear it's not Hadoop 3.2.

How about avoiding this going forward with `hadoop-3`? I doubt we're going to publish builds separately for two different Hadoop 3 versions in the future.

Reply:
> How about avoiding this going forward with `hadoop-3`? I doubt we're going to publish builds separately for two different Hadoop 3 versions in the future.

You mean here we can make profile id as  `hadoop-3`, then we can build spark release with `-Dhadoop.version=3.2.x` and `-Dhadoop.version=3.3.x`?

I think it's ok if we need to maintain two version of hadoop3. Anyway better than `hadoop-3.2`

Reply:
I see, are we going to need a separate Hadoop 3.2 build at all going forward? If not, I think it makes sense to call it hadoop-3.

Reply:
> I see, are we going to need a separate Hadoop 3.2 build at all going forward? If not, I think it makes sense to call it hadoop-3.

In current website, there are only binary file of `hadoop-3.3` & `hadoop-2.7`

Reply:
Right. I don't think we would build with -Dhadoop.version=3.2.x anyway, but someone could try it. It wouldn't mean we need a minor-version-specific Hadoop profile

Reply:
> Right. I don't think we would build with -Dhadoop.version=3.2.x anyway, but someone could try it. It wouldn't mean we need a minor-version-specific Hadoop profile

So make sure, I should continue this pr with profile `hadoop-3`, right?

Reply:
That's my suggestion at least, and it would solve the version confusion we observed in the 3.2.0 release. I don't see a downside.

Reply:
+1 to `hadoop-3`